azure-common/azure/profiles/__init__.py

```diff
@@ -108,6 +108,45 @@ class KnownProfiles(Enum):
         },
         "2017-03-09-profile"
     )
+    v2018_03_01_hybrid = ProfileDefinition(
+        {
+            "azure.keyvault.KeyVaultClient":{
+                None: "2016-10-01"
+            },
+            "azure.mgmt.authorization.AuthorizationManagementClient": {
+                None: "2015-07-01"
+            },
+            "azure.mgmt.compute.ComputeManagementClient": {
+                None: "2017-03-30"
+            },
+            "azure.mgmt.keyvault.KeyVaultManagementClient":{
+                None: "2016-10-01"
+            },
+            "azure.mgmt.network.NetworkManagementClient": {
+                None: "2017-10-01"
+            },
+            "azure.mgmt.storage.StorageManagementClient": {
+                None: "2016-01-01"
+            },
+            "azure.mgmt.resource.policy.PolicyClient": {
+                None: "2016-12-01"
+            },
+            "azure.mgmt.resource.locks.ManagementLockClient": {
+                None: "2016-09-01"
+            },
+            "azure.mgmt.resource.links.ManagementLinkClient": {
+                None: "2016-09-01"
+            },
+            "azure.mgmt.resource.resources.ResourceManagementClient": {
+                None: "2018-02-01"
+            },
+            "azure.mgmt.resource.subscriptions.SubscriptionClient": {
+                None: "2016-06-01"
+            }
+        },
+        "2018-03-01-hybrid"
```

Comment:
do we have any SDK level tests that needs to be updated?

Reply:
I don't have profile specific tests unfortunately :(